app/services/weets.js

```diff
@@ -0,0 +1,6 @@
+const { default: Axios } = require('axios');
+
+exports.getQuote = async () => {
+  const { data } = await Axios.get('https://geek-jokes.sameerkumar.website/api?format=json');
+  return data;
+};
```

Comment:
- Remember to encapsulate all of this in a try-catch so it can handle exceptions.
- You could also add a custom error for when the API fails, you can guide yourself with the ones already in the `error.js` file.
- Remember to use logs to better track functionalities.